client/src/app/home/home.e2e.ts

```diff
@@ -8,8 +8,41 @@ describe('App', () => {
 
   it('should have a title', () => {
     let subject = browser.getTitle();
-    let result  = 'Wertekompass für verteilte und agile Teams';
+    let result = 'Wertekompass für verteilte und agile Teams';
     expect(subject).toEqual(result);
   });
 
+  it('should not accept empty names', () => {
+    let newButton = element(by.css('#btnNewValueCompass'));
```

Comment:
This should be possible to check in the unit tests. See https://angular.io/docs/ts/latest/guide/testing.html.